zqd/packet/packet.go

```diff
@@ -70,6 +72,7 @@ func IngestFile(ctx context.Context, s *space.Space, pcap string) (*IngestProces
 }
 
 func (p *IngestProcess) run(ctx context.Context) error {
+	var minTs, maxTs nano.Ts
```

Comment:
Move this closer to where these are used.

Reply:
@nwt can't do that due to gotos.